server.js

```diff
@@ -5392,7 +5392,7 @@ camp.route(/^\/vscode-marketplace\/(d|v|r)\/(.*)\.(svg|png|gif|jpg|json)$/,
 );
 
 // Eclipse Marketplace integration.
-camp.route(/^\/eclipse-marketplace\/(dt|dm|v|favorites)\/(.*)\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/eclipse-marketplace\/(dt|dm|v|favorites|updated)\/(.*)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Since the github badge calls this `last-update`, how about using the same thing here for consistency?